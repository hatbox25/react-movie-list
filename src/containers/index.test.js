import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '.';

describe('App', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  describe('App Snap Test', () => {
    test('should render correctly', () => {
      expect(toJson(shallow(<App />))).toMatchSnapshot();
    });
  });
});
