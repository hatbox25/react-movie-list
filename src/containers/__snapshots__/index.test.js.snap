// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App Snap Test should render correctly 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <NavigationProvider />
</Provider>
`;
